### PR TITLE
fix: remove unused variables

### PR DESCRIPTION
**Pull request by noraliucode**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
remove unused variables for NFT
- **What is the current behavior?** (You can also link to an open issue here)
unused variables for NFT exist
- **What is the new behavior (if this is a feature change)?**
unused variables for NFT removed
- **Other information**:

### CODE REVIEW

File: src/blockchain/utils.ts [hunk 1]
```diff
@@ -138,7 +135,7 @@ export const capitalizeToken = (symbol: IStableAssets) => {
   return symbol
 }
 
-export const getAbi = (contractName: IContracts) => {
+export const getAbi = (contractName: IContracts | INFTContracts) => {
```
Comment (eswarasai) [inline src/blockchain/utils.ts:138]:
```suggestion
export const getAbi = (contractName: IContracts) => {
```

File: src/blockchain/utils.ts [hunk 2]
```diff
@@ -330,7 +319,7 @@ export const getBancorPoolContract = (
 }
 
 export const getContract = (
-  contractName: IContracts,
+  contractName: IContracts | INFTContracts,
```
Comment (eswarasai) [inline src/blockchain/utils.ts:322]:
```suggestion
  contractName: IContracts,
```

File: src/types/xToken.d.ts [hunk 3]
```diff
@@ -118,16 +114,6 @@ export type IContracts =
   | IXAssetCLR
   | ILendingContracts
   | IXAssetLev
```
Comment (eswarasai) [inline src/types/xToken.d.ts:116]:
```suggestion
  | IXAssetLev
  | INFTContracts
```

File: src/blockchain/utils.ts (hunk 1 above)
Comment (noraliucode) [reply src/blockchain/utils.ts:138]: Fixed!

File: src/blockchain/utils.ts (hunk 2 above)
Comment (noraliucode) [reply src/blockchain/utils.ts:322]: Fixed!

File: src/types/xToken.d.ts (hunk 3 above)
Comment (noraliucode) [reply src/types/xToken.d.ts:116]: Fixed!

**Review by eswarasai — APPROVED** [review]
LGTM 👍

**Pull request merged**